smoke-tests/apps/DetectUnexpectedOtelMetrics/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/DetectUnexpectedOtelMetricsTest.java

```diff
@@ -47,6 +47,22 @@ abstract class DetectUnexpectedOtelMetricsTest {
     EXPECTED_METRIC_NAMES.add("% Of Max Heap Memory Used");
     EXPECTED_METRIC_NAMES.add("GC Total Count");
     EXPECTED_METRIC_NAMES.add("GC Total Time");
+
+    // adding stable metrics from otel.instrumentation.runtime-telemetry
+    EXPECTED_METRIC_NAMES.add("jvm.memory.used"); // heap memory used in bytes
```

Comment:
need to add these metrics name to this PR https://github.com/Azure/azure-sdk-for-java/pull/39960

Reply:
can you add the upstream link for these metrics?

Reply:
I added the metrics to the other PR and put the upstream link here.